tests/roots/test-latex-includegraphics/conf.py

```diff
@@ -0,0 +1,47 @@
+master_doc = 'index'
+
+exclude_patterns = ['_build']
+
+# only to be used on image inclusions with both width and height
```

Comment:
sorry that comment was for earlier version, I then refactored to work with width and/or height. But the index.rst must only contain image inclusions leading to either oversized width or oversized height.